.travis.yml

```diff
@@ -1,58 +1,51 @@
 language: python
-python: 2.7
 sudo: false
+
 cache:
   pip: true
   directories:
     - $HOME/buildout-cache
+
 before_install:
-  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
-  - mkdir $HOME/.buildout
-  - echo "[buildout]" > $HOME/.buildout/default.cfg
-  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
-  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
-  - pip install --upgrade pip setuptools zc.buildout
-  - if [[ `npm -v` != 3.10.7 ]]; then npm i -g npm@3.10.7; fi
-  - npm install -g grunt-cli
-
-env:
-  matrix:
```

Comment:
@loechel the link does not work. As far as I can see we still do not test the different package types. I would also keep in mind the overall execution time of the full test matrix. You usually don't want to run this at all on your local machine.

As I told you during our discussion at the Midsummer sprint, I have some fundamental doubts about using tox on a CI system. There is a fundamental mismatch between a local command like tox and CI which can not be overcome easily (see for instance this tweet from one of the pytest coredevs https://twitter.com/ossronny/status/894442880789557249).

I would suggest that we figure out where we want to go, before putting too much effort into discussions about implementation details. The PLIP is the right place for this discussion, not bobtemplates.plone.